x-pack/test/api_integration/apis/maps/maps_telemetry.ts

```diff
@@ -36,8 +36,8 @@ export default function ({ getService }: FtrProviderContext) {
           return fieldStat.name === 'geo_point';
         }
       );
-      expect(geoPointFieldStats.count).to.be(55);
-      expect(geoPointFieldStats.index_count).to.be(12);
+      expect(geoPointFieldStats.count).to.be(63);
+      expect(geoPointFieldStats.index_count).to.be(13);
```

Comment:
Updated because we added a new attack discovery alerts aad index that maps the ecs fields. Similarly to this https://github.com/elastic/kibana/pull/194322/files#r1781620194 and this https://github.com/elastic/kibana/pull/184541/files#r1628183656.